src/parser/scanner_support.c

```diff
@@ -59,6 +59,13 @@ extern int yycolumn_end;
 
 
 /* matching hint information to output*/
+typedef struct _stmt_hint STMT_HINT;
+struct _stmt_hint
```

Comment:
The two structs `_stmt_hint` and `st_hint_msg` seem to be related, but why is there no `st_` prefix like `st_hint_msg`?

Reply:
Like struct _xxx, starting with a _ is a common practice when the name is not going to be used in the future. Instead, use the typedef name. However, according to your comments, we will make corrections for consistency.